server/sonar-db-dao/src/main/java/org/sonar/db/rule/RuleDefinitionDto.java

```diff
@@ -0,0 +1,347 @@
+    this.status = s;
+    return this;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public RuleDefinitionDto setName(@Nullable String s) {
+    checkArgument(s == null || s.length() <= 255, "Rule name is too long: %s", s);
+    this.name = s;
+    return this;
+  }
+
+  public String getConfigKey() {
+    return configKey;
+  }
+
+  public RuleDefinitionDto setConfigKey(@Nullable String configKey) {
+    this.configKey = configKey;
+    return this;
+  }
+
+  @CheckForNull
+  public Integer getSeverity() {
+    return severity;
+  }
+
+  @CheckForNull
+  public String getSeverityString() {
+    return severity != null ? SeverityUtil.getSeverityFromOrdinal(severity) : null;
+  }
+
+  public RuleDefinitionDto setSeverity(@Nullable String severity) {
+    return this.setSeverity(severity != null ? SeverityUtil.getOrdinalFromSeverity(severity) : null);
+  }
+
+  public RuleDefinitionDto setSeverity(@Nullable Integer severity) {
+    this.severity = severity;
+    return this;
+  }
+
+  public boolean isTemplate() {
+    return isTemplate;
+  }
+
+  public RuleDefinitionDto setIsTemplate(boolean isTemplate) {
+    this.isTemplate = isTemplate;
+    return this;
+  }
+
+  @CheckForNull
+  public String getLanguage() {
+    return language;
+  }
+
+  public RuleDefinitionDto setLanguage(String language) {
+    this.language = language;
+    return this;
+  }
+
+  @CheckForNull
+  public Integer getTemplateId() {
+    return templateId;
+  }
+
+  public RuleDefinitionDto setTemplateId(@Nullable Integer templateId) {
+    this.templateId = templateId;
+    return this;
+  }
+
+  @CheckForNull
+  public String getDefRemediationFunction() {
+    return defRemediationFunction;
+  }
+
+  public RuleDefinitionDto setDefRemediationFunction(@Nullable String defaultRemediationFunction) {
+    this.defRemediationFunction = defaultRemediationFunction;
+    return this;
+  }
+
+  @CheckForNull
+  public String getDefRemediationGapMultiplier() {
+    return defRemediationGapMultiplier;
+  }
+
+  public RuleDefinitionDto setDefRemediationGapMultiplier(@Nullable String defaultRemediationGapMultiplier) {
+    this.defRemediationGapMultiplier = defaultRemediationGapMultiplier;
+    return this;
+  }
+
+  @CheckForNull
+  public String getDefRemediationBaseEffort() {
+    return defRemediationBaseEffort;
+  }
+
+  public RuleDefinitionDto setDefRemediationBaseEffort(@Nullable String defaultRemediationBaseEffort) {
+    this.defRemediationBaseEffort = defaultRemediationBaseEffort;
+    return this;
+  }
+
+  @CheckForNull
+  public String getGapDescription() {
+    return gapDescription;
+  }
+
+  public RuleDefinitionDto setGapDescription(@Nullable String s) {
+    this.gapDescription = s;
+    return this;
+  }
+
+  public Set<String> getSystemTags() {
+    return systemTags == null ? new HashSet<>() : new TreeSet<>(Arrays.asList(StringUtils.split(systemTags, ',')));
+  }
+
+  private String getSystemTagsField() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "getSystemTagsField" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)